### PR TITLE
Fixed the Search bar issue #3917

### PR DESCRIPTION
**Pull request by deepaksinghgusain227**
Fixed Search bar issue #3917

### CODE REVIEW

**Comment by vaishaliwebkul** [comment]
@deepaksinghgusain227 
search key term is removed from search bar when we simply search any product 
https://prnt.sc/uvzhrz

**Comment by vaishaliwebkul** [comment]
Done

**Review by vaishaliwebkul — APPROVED** [review]

File: packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php [hunk 1]
```diff
@@ -29,7 +30,7 @@
                             required
                             name="term"
                             type="search"
-                            value="{{ $term }}"
+                            value="@if(! $image_search ){{ $term }}@endif"
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/layouts/header/index.blade.php:33]: Use use this {{ ! $image_search ?  $term : '' }}.

**Comment by jitendra-webkul** [comment]
Resolved conflicts

**Comment by deepaksinghgusain227** [comment]
Close due to lots of conflicts

**Pull request closed without merging**